### ISSUE TITLE
[Bug] Unable to install Kernal Memory Core without deprecated packages

### ISSUE THREAD

**Issue by 0x59d** [microsoft/kernel-memory#457]
### Context / Scenario

Trying to install Kernel Memory Core package through nuget package manager on dotnet 8 solution. The packages referenced are deprecated and alternative suggestion is the Microsoft.KernelMemory.Core package. 

### What happened?

When attempting to install Microsoft.KernelMemory.Core, I'm receiving the following errors stating unable to find the (deprecated) packages.

Expected: To install Microsoft.KernelMemory.Core without errors/installing deprecated packages

### Importance

I cannot use Kernel Memory

### Platform, Language, Versions

Dotnet 8.0
Windows
Visual Studio 2022
Web API + Console App

### Relevant log output

```shell
Errors:

Error	NU1102	Unable to find package Microsoft.KernelMemory.Abstractions with version (>= 0.50.240502.2)
  - Found 71 version(s) in nuget.org [ Nearest version: 0.40.240430.1 ]
  - Found 0 version(s) in Microsoft Visual Studio Offline Packages	KernelMemory

Error	NU1102	Unable to find package Microsoft.KernelMemory.AI.TikToken with version (>= 0.50.240502.2)
  - Found 2 version(s) in nuget.org [ Nearest version: 0.40.240501.1 ]
  - Found 0 version(s) in Microsoft Visual Studio Offline Packages	KernelMemory	

Error	NU1102	Unable to find package Microsoft.KernelMemory.AI.AzureOpenAI with version (>= 0.50.240502.2)
  - Found 32 version(s) in nuget.org [ Nearest version: 0.40.240501.1 ]
  - Found 0 version(s) in Microsoft Visual Studio Offline Packages	KernelMemory

Full Package Error List:

Microsoft.KernelMemory.MemoryDb.AzureAISearch
Microsoft.KernelMemory.AI.LlamaSharp
Microsoft.KernelMemory.DataFormats.AzureAIDocIntel
Microsoft.KernelMemory.ContentStorage.AzureBlobs
Microsoft.KernelMemory.AI.OpenAI
Microsoft.KernelMemory.Orchestration.RabbitMQ
Microsoft.KernelMemory.MemoryDb.Quadrant
Microsoft.KernelMemory.MemoryDb.Postgres
Microsoft.KernelMemory.Orchestration.AzureQueues
Microsoft.KernelMemory.MemoryDb.MongoDbAtlas
Microsoft.KernelMemory.MemoryDb.Redis
```

**Comment by dluc** [comment]
This should fix the problem https://github.com/microsoft/kernel-memory/pull/458 - New nugets to be published soon

**Comment by 0x59d** [comment]
Cheers, thanks for the promptness

**Comment by dluc** [comment]
Problem fixed, version 0.50.240504.7
